flowzone.yml

```diff
@@ -600,6 +605,11 @@ on:
         type: string
         required: false
         default: ""
+      terraform_projects:
+        description: "Terraform projects to deploy"
+        type: string
+        required: false
+        default: ""
```

Comment:
If this needs to be in a certain format, like JSON, or comma separated, we should specify that in the description.

Be as verbose as possible in the description since it is applied to the readme, and multiple lines are supported.